README.md

```diff
@@ -189,6 +189,17 @@ We use [SemVer](http://semver.org/) for versioning. For the available versions,
 
 ## Upgrade from older versions
 
+### Upgrade to 1.0.0-beta2
+
+The following paragraphs required components have been removed from [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme)
```

Comment:
What is a "paragraphs required component"?

Reply:
As the list below is about composer packages, we should say that:
```suggestion
Dependencies to the following packages have been removed in composer.json:
```
just a proposal, feel free to improve